task-8/src/server.ts

```diff
@@ -1,13 +1,25 @@
 import express from "express";
+import {
+  MikroORM,
+  RequestContext,
+  type EntityManager as IEntityManager,
+} from "@mikro-orm/core";
+import { type PostgreSqlDriver as IPostgreSqlDriver } from "@mikro-orm/postgresql";
 import apiRouter from "./api";
+import ormConfig from "./config/orm.config";
 
 const PORT = 8000;
 
-export const runServer = () => {
+export let entityManager: IEntityManager = {} as IEntityManager;
+
+export const runServer = async () => {
   const app = express();
+  const orm = await MikroORM.init<IPostgreSqlDriver>(ormConfig);
+  entityManager = orm.em;
```

Comment:
question: is entityManager used somewhere?

Reply:
Yea, it is the main import to work with ORM in scope of each of repository.
So basically we create EM once on server initialization, then use it during API calls handling to manipulate DataBase.